### PR TITLE
allow unkeep any keep that is not inactive (active or duplicate)

### PR DESCRIPTION
**Pull request by ahsu1230**
@andrewconner @stkem 

the simple fix for our duplicates problem. The more complex, we can put in lower priority.

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/KeepsCommander.scala [hunk 1]
```diff
@@ -344,7 +344,7 @@ class KeepsCommander @Inject() (
         val (keeps, invalidKeepIds) = db.readWrite { implicit s =>
           val (keeps, invalidKeepIds) = keepIds.map { kId =>
             keepRepo.getByExtIdandLibraryId(kId, libId, excludeState = None) match {
-              case Some(k) if (k.isActive) =>
+              case Some(k) if k.state != KeepStates.INACTIVE =>
                 keepsToFinalize = k +: keepsToFinalize
                 Left(setKeepStateWithSession(k, KeepStates.INACTIVE, userId))
```
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/KeepsCommander.scala:349]: `setKeepStateWithSession` is the part that saves the keep as `INACTIVE`

**Comment by stkem** [comment]
lgtm

**Comment by stkem** [comment]
please make sure it works on kifi.com when this is out

**Comment by ahsu1230** [comment]
yup will do

**Pull request merged**